fix: root folder in treeview drawer not translated EXO-64528

hthabti opened the pull request:
prior to this change, the root folder of treeview drawer wasn't translated. 
after this change it would be well translated

aycherif committed:
TASK-62301: Create FB maintenance and update projects versions/dependencies

sofyenne committed:
fix : share document with user who have left the space - EXO-63717 (#924)

Before this change, when we shared a document with a user who was already a member of the space and then re-shared it after the user had left the space, no symlink would be created on the user's personal document. The problem stemmed from an incorrect check that verified whether the destination user had access to the document to be shared, which always returned true due to the initial share action.
This change is going to check if the owner is a space and the destination user is already a member of this space.

hthabti committed:
fix: root folder in treeview drawer not translated EXO-64528

prior to this change, the root folder of treeview drawer wasn't translated.
after this change it would be well translated

ahamdi commented on documents-webapp/src/main/webapp/vue-app/documents/components/body/views/FolderTreeViewDrawer.vue:
```diff
@@ -25,7 +25,7 @@
               class="body-2 mx-2 mt-1"
               :class="idItemActive === item.id ? 'primary--text font-weight-bold' : ''">
-              {{ item.name }}
+              {{ item.name === 'Documents' ? $t('documents.label.spaceHomeDocuments') : item.name }}               
```
let's avoid expression inside the template if possible, could you add a computed property ?

hthabti replied:
I used method instead, it can't be done through computed (it is an item from array not from data, props..)

sofyenne committed:
fix : Enable to upload a file with name contains a special character from the download menu action - EXO-64549 (#953)

Before this change, we were unable to download a document that contained a '+' character from the download menu action. The problem was that the '+' character was being replaced by a space in the document's download URL. This change will encode the document's download URL.

sofyenne committed:
fix : Redirect user to destination folder after move to or create shortuct document action - EXO-63545 (#959)

This change will redirect the user to the destination folder after moving or creating a shortcut for a document

sofyenne committed:
fix : share document with user who have left the space - EXO-63717 (#945)

hthabti committed:
add method to display item name

ahamdi committed:
Merge branch 'feature/maintenance' into task-64528

Jihed525 committed:
Merge branch 'feature/maintenance' into task-64528

exo-swf commented:
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you
